Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -264,7 +264,7 @@ public class FeaturevisorInstance {
     // MARK: - Statuses
 
     func isReady() -> Bool {
-        return self.statuses.ready
+        return statuses.ready
```

Comment:
in a Swift class instance, are private properties accessible without using `self` or `this` directly?

Reply:
Yes. For Swift it tells in which scope of object you currently are